corehq/apps/export/templates/export/partials/table.html

```diff
@@ -3,7 +3,8 @@
 {% load hq_shared_tags %}
 {% load humanize %}
 
-<table class="table table-striped" data-bind="visible: exports().length">
+<table class="table table-striped"
```

Comment:
I don't agree this improves readability; I'd rather have all the attributes on one line than disrupt the shape of the nesting. I do agree this kind of alignment is helpful when the line has a lot of attributes. But since "a lot of attributes" is fuzzy, I'm fine with chalking that up to stylistic differences.

Reply:
my biggest issue is long lines. long lines are hard for readability due to scrolling and then you really lose your place in the file tbh...

Reply:
yeah, agreed, I think our threshold for "long" is just dufferent